pyproject.toml

```diff
@@ -26,36 +26,35 @@ popcon = 'cfspopcon.cli:run_popcon_cli'
 
 [tool.poetry.dependencies]
 python = ">=3.9,<3.12"
-numpy = "^1.22.4"
-pandas = "^1.4"
-scipy = "^1.8"
-seaborn = "^0.12"
-pyyaml = "^6.0"
-toml = "^0.10.2"
-typing-extensions = "^4.0.1"
-pint = "^0.22"
-xarray = "^2023.4.1"
-pint-xarray = "^0.3"
-ipdb = "^0.13.13"
-click = "^8.1.0"
-netcdf4 = "^1.6.5"
+numpy = ">=1.22.4"
+matplotlib = ">=3.8.2"
+pandas = ">=1.4"
+scipy = ">=1.8"
+pyyaml = ">=6.0"
+typing-extensions = ">=4.0.1"
+pint = ">=0.22"
+xarray = ">=2023.4.1"
+pint-xarray = ">=0.3"
+ipdb = ">=0.13.13"
+click = ">=8.1.0"
+netcdf4 = ">=1.6.5"
 
 [tool.poetry.group.dev.dependencies]
-pre-commit = "^2.20.0"
-black = "^22.10.0"
-pytest = "^7.2.0"
-coverage = "^6.5.0"
-pytest-cov = "^4.0.0"
-types-pyyaml = "^6.0.12.2"
-pandas-stubs = "^1.5.1.221024"
-mypy = "^1.4.1"
-sphinx = "^7.2.6"
-sphinx-rtd-theme = "^2.0.0rc2"
-sphinxcontrib-bibtex = "^2.6.1"
-sphinx-copybutton = "^0.5.2"
-ruff = "^0.0.292"
-nbmake = "^1.4.3"
-nbsphinx = "^0.9.3"
+pre-commit = ">=2.20.0"
+black = ">=22.10.0"
+pytest = ">=7.2.0"
+coverage = ">=6.5.0"
+pytest-cov = ">=4.0.0"
+types-pyyaml = ">=6.0.12.2"
+pandas-stubs = ">=1.5.1.221024"
+mypy = ">=1.4.1"
+sphinx = ">=7.2.6"
+sphinx-rtd-theme = ">=2.0.0rc2"
+sphinxcontrib-bibtex = ">=2.6.1"
+sphinx-copybutton = ">=0.5.2"
+ruff = ">=0.0.292"
+nbmake = ">=1.4.3"
+nbsphinx = ">=0.9.3"
```

Comment:
What motivated all these changes? In general I prefer to have feature changes separate from dependency changes/updates.